Classes/Converter/ExternalConverter.php

```diff
@@ -32,11 +32,11 @@ public function __construct(string $parameters)
     public function convert(string $originalFilePath, string $targetFilePath): void
     {
         $silent = \filter_var(Configuration::get('silent'), FILTER_VALIDATE_BOOLEAN);
-        $command = \sprintf(
-            \escapeshellcmd($this->parameters),
-            CommandUtility::escapeShellArgument($originalFilePath),
-            CommandUtility::escapeShellArgument($targetFilePath)
-        ) . ($silent ? ' >/dev/null 2>&1' : '');
+        $command = ExternalConverter . php\sprintf(
```

Comment:
What exactly is this line supposed to do?
`$command = ExternalConverter . php\sprintf(`